source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -410,6 +413,75 @@ json TransformerInstance::base64url_decode_callback(const inja::Arguments &args)
   return Base64Url::decode(input);
 }
 
+json TransformerInstance::word_count_callback(const inja::Arguments &args) {
+  return std::to_string(json_word_count(args.at(0)));
```

Comment:
do we need the to_string() here? i.e. is there a disadvantage of returning an int?